.changeset/small-eagles-try.md

```diff
@@ -0,0 +1,19 @@
+---
+'@sap-ux/odata-cli': minor
+'@sap-ux/generator-simple-fe': minor
+'@sap-ux/axios-extension': minor
+'@sap-ux/backend-proxy-middleware': minor
+'@sap-ux/btp-utils': minor
+'@sap-ux/fe-fpm-writer': minor
+'@sap-ux/fiori-elements-writer': minor
+'@sap-ux/fiori-freestyle-writer': minor
+'@sap-ux/logger': minor
+'@sap-ux/odata-service-writer': minor
+'@sap-ux/store': minor
+'@sap-ux/ui5-application-writer': minor
+'@sap-ux/ui5-config': minor
+'@sap-ux/ui5-proxy-middleware': minor
+'@sap-ux/yaml': minor
```

Comment:
Not sure if this is technically a major change as it is a breaking change to drop node 12?